app/src/main/java/ch/epfl/sweng/SDP/home/LeaderboardActivity.java

```diff
@@ -70,26 +72,35 @@ protected void onCreate(Bundle savedInstanceState) {
 
         final EditText searchField = findViewById(R.id.searchField);
```

Comment:
Add comments and try reduce the size of this function

Reply:
ok